pages/_app.tsx

```diff
@@ -27,4 +27,4 @@ function MyApp({ Component, pageProps }: AppPropsWithLayout) {
   );
 }
 
-export default MyApp;
+export default appWithTranslation<never>(MyApp);
```

Comment:
Can we add a proper type here instead of `never`?

Reply:
I think there is an issue related to that .
https://github.com/i18next/next-i18next/issues/1944

Reply:
Noted.